src/index.html

```diff
@@ -11,7 +11,185 @@
+
+      <fieldset class="fieldset-block">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="15"
+            required
+            class="input-margin"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="catlover"
+              value="yes"
+              class="input-margin"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="catlover"
+              value="no"
+              class="input-margin"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="color"
+            class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            class="input-margin"
+            name="carbrands"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How can you rate our work?
+          <input
+            type="range"
+            name="rate"
+            class="input-margin"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-block">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="feedback"
+            rows="2"
+            cols="20"
+            class="input-margin"
+         ></textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend" class="input-margin">
+            <option value="y">Yes</option>
+            <option value="n">No</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button type="submit">
+        Submit
+      </button>
+      
+    </form>
```

Comment:
```suggestion
      </button>
    </form>
```
Don't leave empty lines between parent and child elements